content/topics/redux-thunks/_index.md

```diff
@@ -6,7 +6,17 @@ prerequisites:
   - topics/redux-intro
   soft: []
 title: Redux Thunks
+ready: True
 ---
 
+Redux Thunk is a really confusing thing when you first hear about it.especailly that word “thunk.” So let’s clear that up first.
+
+Redux Thunk middleware allows you to write action creators that return a function instead of an action. The thunk can be used to delay the dispatch of an action, or to dispatch only if a certain condition is met,so instead of performing some work now, you produce a function body or unevaluated expression (the “thunk”) which can optionally be used to perform the work later.
```

Comment:
You copy-pasted this from the docs.  Why is this a useful paragraph? Does it add any value to the reader?

What are thunks for?  Why would you choose to use thunks in an application? Try to tell people why they should care.

Eg: 
" When building a redux application, you often get to a point where xxx. That's when thunks come in. Thunks allow you to yyy"